### ISSUE TITLE
New Rule: no-computed-arrows

### ISSUE THREAD

**Issue by nathanhammond** [ember-cli/eslint-plugin-ember#263]
```js
// BAD
const myObject = Ember.Object.extend({
  dependentKey: 'hello',

  myComputed: Ember.computed('dependentKey', () => {
    this.get('dependentKey'); // Whoops.
  })
});

// GOOD
const myObject = Ember.Object.extend({
  dependentKey: 'hello',

  myComputed: Ember.computed('dependentKey', function() {
    this.get('dependentKey'); // No problem.
  })
});
```

This is not typically a problem in the first pass (if the code doesn't work the user will not commit it), but it can create a surprising refactoring hazard that is not easy to identify in code review.

**Comment by kellyselden** [comment]
If someone does implement this, please check the source module of the computed. Arrow functions are perfectly fine using computed from ember-macro-helpers or ember-awesome-macros.
